charts/xp-treatment/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
         imagePullPolicy: {{ .Values.deployment.image.pullPolicy }}
         env:
           {{- with .Values.deployment.extraEnvs }}
-          {{- toYaml . | nindent 12 }}
```

Comment:
12 should still work too, right?

Reply:
Hmm yes good point, now that I checked again it seems like having an overindented list doesn't make the yaml invalid 🤔 
![Screenshot 2023-07-26 at 2 11 23 PM](https://github.com/caraml-dev/helm-charts/assets/36802364/ef1229a4-3701-40c3-b661-7b03460de5f0)

Okay let me go back and dig this further

Reply:
Closing this issue because it's probably not fixing anything.